spec/src/main/asciidoc/chapters/introduction/entities.asciidoc

```diff
@@ -0,0 +1,142 @@
+== Type system
+
+This specification assumes that data is represented in a structured form, that is, that it is expressed in terms of:
+
+1. **atomic values**,
+2. **collections**, and
+3. **structures**.
+
+Any atomic value, collection, or structure belongs to at least one well-defined type.
+
+=== Atomic values
+
+_Atomic values_ are things defined externally to this specification, taken to include--at bare minimum--strings of Unicode characters, integers, floating point numbers, and dates, times, and datetimes.
+Every atomic value has a named type, referred to in the sister-specifications of this specification as a _basic type_.
```

Comment:
"Atomic values" is fine, but given that they are referred to as basic types in the other specifications, would it make sense to use the term "basic values" here to better align with the other specs?